index.js

```diff
@@ -43,6 +43,95 @@
 //.
 //. Sanctuary is designed to work in Node.js and in ES5-compatible browsers.
 //.
+//. ## Ramda
+//.
+//. [Ramda][] provides several functions which return problematic values
+//. such as `undefined`, `Infinity`, or `NaN` when applied to unsuitable
+//. inputs. These are known as [partial functions][]. Partial functions
+//. necessitate the use of guards or null checks. In order to safely use
+//. `R.head`, for example, one must ensure that the array is non-empty:
+//.
+//.     if (R.isEmpty(xs)) {
+//.       // ...
+//.     } else {
+//.       return f(R.head(xs));
+//.     }
+//.
+//. Using the Maybe type renders such guards (and null checks) unnecessary.
+//. Changing functions such as `R.head` to return Maybe values was proposed
+//. in [ramda/ramda#683][], but was considered too much of a stretch for
+//. JavaScript programmers. Sanctuary was released the following month,
+//. in January 2015, as a companion library to Ramda.
+//.
+//. In addition to broadening in scope in the years since its release,
+//. Sanctuary's philosophy has diverged from Ramda's in several respects.
+//.
+//. ### Totality
+//.
+//. Every Sanctuary function is defined for every value which is a member of
+//. the function's input type. Such functions are known as [total functions][].
+//. Ramda, on the other hand, contains a number of [partial functions][].
+//.
+//. ### Information preservation
+//.
+//. Certain Sanctuary functions preserve more information than their Ramda
+//. counterparts. For example, one can determine the length of `xs` from
+//. the result of `S.tail(xs)` but not from the result of `R.tail(xs)`
+//. (since `R.tail([])` evaluates to `[]`).
```

Comment:
Maybe just:

```diff
- since `R.tail([])` evaluates to `[]`
+ `R.tail([[]])` equals `R.tail([])`, but `S.tail([[]])` does not equal `S.tail([])`
```